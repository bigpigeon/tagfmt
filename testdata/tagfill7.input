//tagfmt -f "json=or(:tag,snake(:field))"

package main

type OrderDetail struct {
	ID       string   ``
	UserName string   `json:"name"`
	OrderID  string   `json:"order"`
	Callback string   ``
	Address  []string ``
}
