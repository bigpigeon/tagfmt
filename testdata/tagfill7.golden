//tagfmt -f "json=or(:tag,snake(:field))"

package main

type OrderDetail struct {
	ID       string   `json:"id"`
	UserName string   `json:"name"`
	OrderID  string   `json:"order"`
	Callback string   `json:"callback"`
	Address  []string `json:"address"`
}
